t/stdout.t

```diff
@@ -151,9 +151,35 @@ status: nil
     }
 --- response_body_like chomp
 \Aok: nil
-stdout: 
-stderr: (?:|he
-)
-reason: failed to read stdout: timeout
+stdout: ''
+stderr: '(?:|he
+)'
+reason: failed to wait process: timeout
 status: nil
 \z
+
+
+
+=== TEST 6: clean up the sub-process when failed to read
```

Comment:
more like "when failed to wait"? We could read, but the timeout was thrown by `wait()`, not `read()`.